sdk/core/azure-core-metrics-opentelemetry/src/main/java/com/azure/core/metrics/opentelemetry/Utils.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.metrics.opentelemetry;
+
+import com.azure.core.util.AttributesBuilder;
+import com.azure.core.util.Context;
+import com.azure.core.util.logging.ClientLogger;
+import io.opentelemetry.api.common.Attributes;
+
+import java.util.Optional;
+
+import static com.azure.core.util.tracing.Tracer.PARENT_TRACE_CONTEXT_KEY;
+
+class Utils {
```

Comment:
nit. I'd suggest more specific name. Even though it's not public it shows up in the IDE searches/autocomplete for our devs.